lib/fathead/jmeter_docs/parse.js

```diff
@@ -0,0 +1,164 @@
+                    console.log("** JMeter Fathead Processing Completed at: " + new Date());
+
+                });
+        }
+
+        console.log("** JMeter Fathead Processing Started at: " + new Date());
+        let $ = cheerio.load(fs.readFileSync(DOC_BASE_FILE));
+        let allUrls = [];
+        $('.overviewSummary td.colFirst > a').each(function(i, elem) {
+            allUrls[i] = $(this).attr("href");
+            console.log(allUrls[i]);
+        });
+        console.log("Total urls to process: " + allUrls.length);
+        urlFetcher(allUrls);
+    }
+
+    // function to parse all php functions
+    function parserFunction(url, htmlData) {
+        let $ = cheerio.load(htmlData);
+        $('li.blockList > table').each(function(i, elem) {
+            //console.log($(this).html());
+            let children = $(this).children();
+            // extract category
+            let category = children.eq(0).children().eq(0).text().replace(/Summary|Types/g, '').trim();
+            // Process each row of the Category type
+            children.eq(2).children().each(function(i, elem) {
+
+                // Process Type name and cleanup <T>'sand remove "."
+                let nameRowElement = $(this).children().eq(0);
+                let nameRowElementText = nameRowElement.text().replace(/<.*>/g, '');
+                nameRowElementText = nameRowElementText.split(".").join(" ");
+                // extract detail url for the current type
+                let nameRowElementShowMoreUrl = nameRowElement.find('a').attr('href');
+                nameRowElementShowMoreUrl = BASE_URL + nameRowElementShowMoreUrl.replace(/^(?:\.\.\/)+/, "")
+
+                // Removing dup with same class and interface name
+                if (dupMap[nameRowElementText]) {
+                    nameRowElementText+= url.split("/").slice(0, -1).join(' ');
+                }else{
+                    dupMap[nameRowElementText] = category;
+                }
+                // Process details of the category
+                let detailRowElement = $(this).children().eq(1).children().eq(0);
+
+                detailRowElement.find('a').each(function(i, elem) {
+                    // Removing <a> tags for now.
+                    $(this).replaceWith($(this).html());
+                });
+
+                let details = '';
+                if(detailRowElement.html()){
+                    //console.log(detailRowElement.html());
+                    details = detailRowElement.html();
+                    details = details.replace(/\\/g,'\\\\').replace(/\n/g, ' ').replace(/\\x00/g, '\\\\x00');
+                }
+                let detailNode = "<section class='prog__container'>"
+                                + details
+                                + "</section>"
+                const item = {
+                    name: nameRowElementText,
+                    abstract: detailNode,
+                    categories: category,
```

Comment:
The `category` needs to be prefixed wit `JMeter` (to ensure its unique and specific), and should be pluralized. E.g. `JMeter Classses`, `JMeter Errors`

that way when a user searches "jmeter errors" we will show them the list of topics in that category 👍